code/modules/mob/living/carbon/metroid/metroid.dm

```diff
@@ -925,6 +925,50 @@ mob/living/carbon/slime/var/temperature_resistance = T0C+75
 	if(!uses)
 		qdel(src)
 
+/obj/item/stack/tile/bluespace
+	name = "bluespace floor tile"
+	singular_name = "floor tile"
+	desc = "Through a series of micro-teleports these tiles let people move at incredible speeds"

```

Comment:
comma between teleports and these
"Through a series of micro-teleports, these tiles let people move at incredible speeds."
... And a period